azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeSupportedScreenOrientation.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+import android.content.pm.ActivityInfo;
+
+public enum CallCompositeSupportedScreenOrientation {
```

Comment:
We should not create enums for public java API.
Use ExpandableStringEnum class.

Reply:
In my understanding, expandable enum is used when we provide option to Contoso to add their own custom enums? 
For orientations, we do not want that. We do not want contoso to set custom enums that will make a orientation which is not supported or that will break the UI.
So I chose Java Enums for this.

Reply:
No, we use it not to let Contoso to extend, it is for us to be able update this enum with more options.

Reply:
The API for ExpandableEnum `fromString()` is public and has to be public.
For example, the error codes, contoso can build a new error code of type CallCompositeErrorCode. 
i.e. val error = CallCompositeErrorCode.fromString("ARSENAL LOST THE LEAGUE")

So this allows contoso to extend I think.

Reply:
Updated with ExpandableString